Classes/Resource/Handler/ImagickResource.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace IchHabRecht\Filefill\Resource\Handler;
+
+use IchHabRecht\Filefill\Resource\RemoteResourceInterface;
+use TYPO3\CMS\Core\Resource\FileInterface;
+
+class ImagickResource implements RemoteResourceInterface
+{
+    /**
+     * @var array
+     */
+    protected array $allowedFileExtensions = [
+        'gif',
+        'jpeg',
+        'jpg',
+        'png',
+        'svg',
```

Comment:
I would opt to remove this extension from the list of supported extensions. As far as I know it can't be handled by ImageMagick, can it?